### PR TITLE
Fixed #30258 -- Adjusted postgres schema value quoting of ranges.

### PR DESCRIPTION
**Pull request by charettes**
https://code.djangoproject.com/ticket/30258

Thanks Tilman Koschnick for the report and patch.

### CODE REVIEW

**Comment by charettes** [comment]
The test failures seems to be caused by a recent release of `unittest-xml-reporting`.

**Comment by charettes** [comment]
Actual commit that broke it https://github.com/xmlrunner/unittest-xml-reporting/commit/4890cd7ca3ccfcd8fc881f0cdd1d257efe5f5e5f#r32781252

File: tests/postgres_tests/test_constraints.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+try:
+    from psycopg2.extras import NumericRange
+except ImportError:
+    pass
+
+
+class SchemaTests(PostgreSQLTestCase):
+    def get_constraints(self, table):
+        """
```
Comment (timgraham) [inline tests/postgres_tests/test_constraints.py]: Seems to fine on a single line.

Comment (charettes) [reply tests/postgres_tests/test_constraints.py]: Will change, FWIW this was extracted as is from `test_indexes`

https://github.com/django/django/blob/c18573187ad160d7c9275173e0082863a836648e/tests/postgres_tests/test_indexes.py#L143-L148

File: tests/postgres_tests/test_constraints.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+class SchemaTests(PostgreSQLTestCase):
+    def get_constraints(self, table):
+        """Get the constraints on the table using a new cursor."""
+        with connection.cursor() as cursor:
+            return connection.introspection.get_constraints(cursor, table)
+
+    def test_check_constraint_range_value(self):
+        self.assertNotIn('field', self.get_constraints(RangesModel._meta.db_table))
```
Comment (felixxm) [inline tests/postgres_tests/test_constraints.py]: I think this assertion has been copied from another test. I would use
```python
constraint_name = 'ints_between'
self.assertNotIn(constraint_name, self.get_constraints(RangesModel._meta.db_table))
```

**Pull request merged**